### PR TITLE
New functions for adding and updating packed variables 

### PR DESCRIPTION
**Pull request by dehann**
Adding `addVariablePacked` and `updateVariablePacked`.

## Test code
```julia
using Revise
using NavAbilitySDK
# using NavAbilityCaesarExtensions
using IncrementalInference
using RoME
using DistributedFactorGraphs

memFg = initfg()
v = IncrementalInference.addVariable!(memFg, :x0, Pose2)
v = IncrementalInference.addVariable!(memFg, :x1, Pose2)


client = NavAbilityHttpsClient()
context = Client(
    "guest@navability.io",
    "TestAdd",
    "TestAdd")

NavAbilitySDK.ls(client, context) |> fetch

# Add the variable

# Add a new one
packedVariable = packVariable(memFg, v)
eventId = addVariablePacked(client, context, packedVariable) |> fetch
NavAbilitySDK.getStatusLatest(client, eventId) |> fetch

# Update the existing one

v = NavAbilitySDK.getVariable(client, context, "x1") |> fetch
push!(v["tags"], "TEST")

eventId = updateVariablePacked(client, context, v) |> fetch
NavAbilitySDK.getStatusLatest(client, eventId) |> fetch
```

### CODE REVIEW

File: src/navability/services/Variable.jl [hunk 1]
```diff
@@ -41,6 +41,31 @@ function addVariable(navAbilityClient::NavAbilityClient, client::Client, variabl
     return @async addPackedVariable(navAbilityClient, client, variable)
 end
 
+function addPackedVariableEvent(client::NavAbilityClient, variable::Dict; options=Dict{String, Any}())::String
+    response = client.mutate(MutationOptions(
+        "addVariablePacked",
```
Comment (jim-hill-r) [inline src/navability/services/Variable.jl:46]: bug is here:
```suggestion
        "sdk_add_variable_packed",
```

File: src/NavAbilitySDK.jl [hunk 2]
```diff
@@ -73,7 +74,7 @@ include("./navability/services/StandardAPI.jl")
 include("./navability/services/DataBlobs.jl")
 include("./navability/services/Session.jl")
 export getVariable, getVariables, listVariables, ls
-export addVariable, addPackedVariable
+export addVariable, addPackedVariable, addPackedVariableNew, updatePackedVariableNew
```
Comment (jim-hill-r) [inline src/NavAbilitySDK.jl:77]: It's all good, but why add a new endpoint. The goal is to deprecate the old endpoint, so SDK should only call the new and remove old references.

I would have changed the old implementation to addPackedVariableOld and pointed existing function to the new endpoint. This moves the needle toward deprecation but updating all existing users of the SDK to new versus having two forks we now need to maintain.

**Review by jim-hill-r — CHANGES_REQUESTED** [review]

File: src/NavAbilitySDK.jl (hunk 2 above)
Comment (GearsAD) [reply src/NavAbilitySDK.jl:77]: Fair, I didn't know whether how we were planning on deprecating the older calls. If that works then yes it's better.

Comment (GearsAD) [reply src/NavAbilitySDK.jl:77]: I mean, if the signatures are the same, then we don't even need to deprecate the old ones.

Comment (jim-hill-r) [reply src/NavAbilitySDK.jl]: I believe in SDK there should be no need to deprecate.. just replace. In cloud I run both until all sdks are sorted.

Comment (jim-hill-r) [reply src/NavAbilitySDK.jl]: I was suggesting old just in case u want it in the event something was acting odd. We know the old is stable.

Comment (GearsAD) [reply src/NavAbilitySDK.jl]: Went with a middle line approach: new functions are there, but standard API uses old function, we can change that as we test it out.

**Review by jim-hill-r — APPROVED** [review]

File: src/navability/services/Variable.jl (hunk 1 above)
Comment (dehann) [reply src/navability/services/Variable.jl]: every time, thanks.. sigh

**Pull request merged**